content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    frequencies_range = np.linspace(center_freq + 240e3, center_freq + 260e3, 41)
</pre><p>Why this scans only on positive side? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
@nkanazawa1989 
sorry for the late reply.
This is because the range is narrowed down because the noise is great overall.

This is the result of an experiment in the frequency domain below.
```
center_freq = backend_defaults.meas_freq_est[qubit]  # an estimate for the resonator frequency
freq_span = 0.2e6 # resonator scan span. The span should be larger than the resonator linewidth kappa
frequencies_range = np.linspace(center_freq-freq_span/2, center_freq+freq_span/2, 41)
```

![image](https://user-images.githubusercontent.com/12453501/217479478-c651860d-4a6d-492b-88a3-4139a6fd9b1b.png)

This is the result of an experiment in the frequency positive side
```
center_freq = backend_defaults.meas_freq_est[qubit]  # an estimate for the resonator frequency
freq_span = 0.2e6 # resonator scan span. The span should be larger than the resonator linewidth kappa
frequencies_range = np.linspace(center_freq + 240e3, center_freq + 260e3, 41)
```

![image](https://user-images.githubusercontent.com/12453501/217479936-405eba13-29c7-4834-be04-62643e7a0f06.png)

This is the result of an experiment in the frequency negative side
```
center_freq = backend_defaults.meas_freq_est[qubit]  # an estimate for the resonator frequency
freq_span = 0.2e6 # resonator scan span. The span should be larger than the resonator linewidth kappa
frequencies_range = np.linspace(center_freq - 260e3, center_freq - 240e3, 41)
```
![image](https://user-images.githubusercontent.com/12453501/217480355-6eef54a4-24e7-49cf-b2f0-da742722788c.png)

I judged that the positive side has a peak. However, it cannot be denied that it is arbitrary.

Reply:
I was running with amp=0.24. It may be too big.